integration-tests/messaging/src/test/java/org/apache/camel/quarkus/component/messaging/it/ActiveMQTestResource.java

```diff
@@ -43,6 +45,13 @@
         try {
             container = new GenericContainer(ACTIVEMQ_IMAGE)
                     .withExposedPorts(ACTIVEMQ_PORT)
+                    .withLogConsumer(new Consumer<OutputFrame>() {
+                        @Override
+                        public void accept(OutputFrame outputFrame) {
+                            System.out.println(outputFrame.getUtf8String());
```

Comment:
Maybe this should be replaced by slf4j ?

Reply:
I can remove the logging altogether if you'd prefer. It's just useful for debugging if we hit issues.

Reply:
I meant [this one](https://github.com/testcontainers/testcontainers-java/blob/master/core/src/main/java/org/testcontainers/containers/output/Slf4jLogConsumer.java) that should properly handle the output frame.

Reply:
> I can remove the logging altogether if you'd prefer. It's just useful for debugging if we hit issues.

No worries we can keep it, was just wondering if we can make it gogin to the logger so we can control it from properties, but not a huge issue

Reply:
Latest commit uses `Slf4jLogConsumer` :+1:

Reply:
does it work as advertised ?

Reply:
Works ok. Everything gets prefixed with `STDOUT:`, so at least it's clear what's coming from the container and what isn't.

Reply:
Ah yeah, that issue ... I think I did implement my own version some time ago, maybe I'll copy over this repo, if only I could remember where it it